### PR TITLE
Use PNG format for inline notebook plot 

### PR DESCRIPTION
**Pull request by jschueller**
Avoids to send a huge svg file when the data is big:
http://trac.openturns.org/ticket/944

### CODE REVIEW

**Comment by jschueller** [comment]
SVG is still cool (no pixels, always right size), maybe I should try to render to svg first and if the length of the string exceeds a limit fallback to png. Thoughts ?

**Comment by jschueller** [comment]
It is not possible to check the length, and the generation of the svg itself is costly too, so this is the best option.

**Comment by regislebrun** [comment]
+1

**Comment by jschueller** [comment]
@regislebrun I added a resourcemap key to switch png/svg (still defaults to png)

File: python/src/Graph.i [hunk 1]
```diff
@@ -3,58 +3,32 @@
...
   {
-    typedef _PySequence_ Type;
+    typedef _PyObject_ Type;
   };
 
   template <>
   inline
   OT::Graph
-  convert<_PySequence_,OT::Graph>(PyObject * pyObj)
+  convert<_PyObject_, OT::Graph>(PyObject * pyObj)
```
Comment (dbarbier) [inline python/src/Graph.i]: Is this block still needed?  It looks like it could be dropped.

Comment (jschueller) [reply python/src/Graph.i]: Yes

**Comment by regislebrun** [comment]
still +1

**Pull request merged**